### ISSUE TITLE
network,bluetooth and notification icons aren't correct

### ISSUE THREAD

**Issue by ghost** [B00merang-Artwork/Windows-XP#9]
![image](https://user-images.githubusercontent.com/37030489/87689660-bba27300-c79d-11ea-8159-84ea1b9a9b63.png)

**Comment by Elbullazul** [comment]
do you have examples of what those should look like?

**Comment by ghost** [comment]
> do you have examples of what those should look like?

Yes they are already included in your icon package, but they are not being used
![image](https://user-images.githubusercontent.com/37030489/88002093-ea5e7780-cb16-11ea-8288-fcf341f62880.png)

same as Bluetooth 
![image](https://user-images.githubusercontent.com/37030489/88002360-9902b800-cb17-11ea-9dc4-637602aa7694.png)

I'm not sure about notifications

edit: network icon is fine when disconnected or internet not available, the problem is only when network is connected and internet is available

**Comment by Elbullazul** [comment]
can you provide details about your desktop environment (name & version) and distro?

**Comment by ghost** [comment]
> can you provide details about your desktop environment (name & version) and distro?

I use cinnamon 4.6.6 on Arch Linux
